src/UI/examples/Card/RepositoryObject/with_object_icon_and_actions.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/* Copyright (c) 2018 Jesús López <lopez@leifos.com> Extended GPL, see docs/LICENSE */
+
+function with_object_icon_and_actions() {
+	//Init Factory and Renderer
+	global $DIC;
+	$f = $DIC->ui()->factory();
+	$renderer = $DIC->ui()->renderer();
+
+	$icon = $f->icon()->custom(ilUtil::getImagePath("icon_crs.svg"), 'Course', 'responsive');
```

Comment:
Why would you use a custom icon here? Note that course is a standard icon. Take it from the standard set please.